package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "swagger-nodegen-cli",
-  "version": "2.4.4",
-  "description": "A simple convenience package that wraps the swagger-codegen-cli-2.4.4 for the node/npm environment.",
+  "version": "2.4.5-mitel.1",
+  "description": "A simple convenience package that wraps the swagger-codegen-cli-2.4.5-mitel.1 for the node/npm environment.",
```

Comment:
Description shouldn't have the specific version listed.  We're getting that out of the package version pairing.